typescript/src/supernova.d.ts

```diff
@@ -287,17 +287,23 @@ declare global {
     type: DocumentationItemType
     slug: string
     userSlug: string | null
+    configuration: {
+      isHidden: boolean
+      isPrivate: boolean
+    }
   }
 
   type DocumentationGroup = DocumentationItem & {
+    type: "Group"
```

Comment:
Adding type specifically to the doc group and page. It will help us with type checking so when we check if the item has type group typescript will automatically knows that it is `DocumentationGroup` and we don't need to type cast it